src/scss/cdb-components/_buttons.scss

```diff
@@ -305,6 +305,12 @@ Layout Component:
   &:hover {
     border-color: $cBlue;
   }
+
+  &.is-disabled {
+    &:hover {
+      border-color: $cHintText;
```

Comment:
shouldn't it be 
```     border-color: $cBlue;
```

Reply:
No, in this button we change the border in `hover`, so when it's disabled we have to keep the same color

Reply:
I'm blind